src/tests/ftest/scrubber/snapshot.py

```diff
@@ -0,0 +1,51 @@
+"""
+  (C) Copyright 2018-2023 Intel Corporation.
+
+  SPDX-License-Identifier: BSD-2-Clause-Patent
+"""
+import time
+
+from scrubber_test_base import TestWithScrubber
+
+
+class TestScrubberEvictWithSnapshot(TestWithScrubber):
+    # pylint: disable=too-many-nested-blocks
+    """Inject Checksum Fault with scrubber enabled
+    and scrubber threshold set to a certain value.
+    Snapshot taken while checksum faults are injected.
+
+    :avocado: recursive
+    """
+    def test_target_eviction_during_snapshot(self):
+        """JIRA ID: DAOS-7333
+
+        - Take a container snapshot while the csum faults
+        are generated and target is evicted eventually.
+
+        :avocado: tags=all,pr,daily_regression
```

Comment:
pr and daily?

Reply:
All these tests need to be moved to full_regression. Forgot to change it... We have coverage for scrubber under pr and daily regressions.

Reply:
Moved to full_regression.